### PR TITLE
update audit text as agreed in the WG meeting

### PR DESCRIPTION
**Pull request by christ1ne**

### CODE REVIEW

File: inference_rules.adoc [hunk 1]
```diff
@@ -115,6 +115,19 @@ input dataset in any form.
...
+
+The auditor should not have the conflict of interest.
+The relavant software, documentation and testing results will be provided to the auditor under NDA.
+The audit is only for available category.
+
+If you fail the audit to a material level, your result will be moved to open or removed based review committee decision.
+We will select two entries to audit per round. One entry will be randomly chosen, one entry will be based on review committee decision. One entry here means a submitter-hardware combination.
+
+The burden is on the submitter to provide to the auditor under NDA sufficient materials to demonstrate that the submission is compliant with the rules. The auditor will submit to the review committee a report describing the work that was performed, a list of unresolved issues, and a recommendation on whether the submission is compliant. 
+
```
Comment (DilipSequeira) [inline inference_rules.adoc:130]: I would like to resequence some of the material here, clarify some issues around how the submissions are chosen, and have some minor copy edits. I can break it down line by line if it helps, but how about:

=== Audit Process

In each round, two Available submissions will be chosen for audit: one selected by the review committee, and one at random from all submissions. A "submission" shall denote a combination of a submitter and a platform (equivalent to a line in the results table) and the randomly chosen submission shall be selected from all submissions with equal probability. 

An auditor shall be chosen by the review committee who has no conflict of interest with the submitter. 

The burden is on the submitter to provide sufficient materials to demonstrate that the submission is compliant with the rules. Any such materials, including software, documentation, testing results and machine access will be provided to the auditor under NDA. 

The submitter shall provide two days of hardware access, at a time mutually agreed with the auditor. The first day will be used to run a pre-agreed list of tests, and to verify other system parameters if needed. The second day will allow the auditor to run additional tests based on outcome of the first day. 

The audit will ideally conclude before publication, but in any case, no more than 30 days after publication.  The auditor shall submit to the review committee a report describing the work that was performed, a list of unresolved issues, and a recommendation on whether the submission is compliant. 

Submissions that fail the audit at a material level will be moved to open or removed, by review committee decision.

Comment (christ1ne) [reply inference_rules.adoc:130]: LGTM except for 2 things:
1) two Available submissions  -> two closed Available submissions
2) how about adding the following: The process of random selection has two stages: 1) randomly pick one submitter. 2) randomly pick one system from that submitter from 1)

Comment (DilipSequeira) [reply inference_rules.adoc:130]: Thanks - revised version:

=== Audit Process

In each round, two submissions will be audited: one selected by the review committee, and one at random from all submissions. A "submission" for audit purposes shall denote a combination of a submitter and a platform (equivalent to a line in the results table). Only Available submissions in Closed division are auditable. 

The process of random selection is in two stages: first a submitter is randomly chosen from all submitters with auditable submissions, then one of those submissions is randomly chosen.

An auditor shall be chosen by the review committee who has no conflict of interest with the submitter.

The burden is on the submitter to provide sufficient materials to demonstrate that the submission is compliant with the rules. Any such materials, including software, documentation, testing results and machine access will be provided to the auditor under NDA.

The submitter shall provide two days of hardware access, at a time mutually agreed with the auditor. The first day will be used to run a pre-agreed list of tests, and to verify other system parameters if needed. The second day will allow the auditor to run additional tests based on outcome of the first day.

The audit will ideally conclude before publication, but in any case, no more than 30 days after publication. The auditor shall submit to the review committee a report describing the work that was performed, a list of unresolved issues, and a recommendation on whether the submission is compliant.

Submissions that fail the audit at a material level will be moved to open or removed, by review committee decision.

**Comment by christ1ne** [comment]
Thanks! I will update the PR to reflect this updated text. We should close it in the next inf meeting.

**Comment by christ1ne** [comment]
@tjablin

**Comment by christ1ne** [comment]
WG approved

**Pull request merged**